### PR TITLE
Code The Dream Assignment   

### PR DESCRIPTION
**Pull request by curt4207**

### CODE REVIEW

File: src/AddTodoForm.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+function AddTodoForm(){
+    return(
+     <form>
+     <label htmlFor="todoList"> Title:
```
Comment (Just-Bob-Dev) [inline src/AddTodoForm.js]: These should be indented within their parent <form> element.

File: src/App.js [hunk 2]
```diff
@@ -1,44 +1,35 @@
...
+
 
 
 function App() {
 
   return (
     <div>
       <h1>{title}</h1>
+      <AddTodoForm/>
+    <TodoList/>
```
Comment (Just-Bob-Dev) [inline src/App.js]: <TodoList/> and <Search /> should also be indented.

File: src/TodoList.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+const todoList = [
+    {
+    id: 1,
+    title: "What to Cook",
+    objectID: 0,
```
Comment (Just-Bob-Dev) [inline src/TodoList.js]: formatting

File: src/AddTodoForm.js (hunk 1 above)
Comment (Just-Bob-Dev) [reply src/AddTodoForm.js]: Also the id="todoTitle"

**Review by Just-Bob-Dev — APPROVED** [review]
Everything works the way its supposed to. You just have a couple of spots that could benefit from more formatting. Also naming of the id and htmlFor in your addTodoForm component.

**Comment by Just-Bob-Dev** [comment]
Hey Curt, while reviewing your PR I noticed your still on the lesson-1-1 branch. After you get this merged in don't forget to create a new branch for next weeks work.

File: src/AddTodoForm.js (hunk 1 above)
Comment (curt4207) [reply src/AddTodoForm.js]: thx, working on it today.

File: src/AddTodoForm.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+const AddTodoForm = (props) => {
+    const handleAddTodo = (event) => {
+        event.preventDefault();
+        console.log();
```
Comment (Just-Bob-Dev) [inline src/AddTodoForm.js:6]: Empty console.log

File: src/AddTodoForm.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+        const todoTitle = event.target.title.value;//retrieve the value of the `title` element from the event target and store it in a variable.
+        console.log(todoTitle);
+        event.target.reset();
+        props.onAddTodo(todoTitle);
+        
+    };
+    return(
+        <form onSubmit={handleAddTodo}>
+        <label htmlFor="todoTitle"> Title:
```
Comment (Just-Bob-Dev) [inline src/AddTodoForm.js:15]: This area could use a bit of formatting.

File: src/todoListItem.js [hunk 6]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import TodoList from "./TodoList";
+
+const TodoListItem = (props) => {
+    return(
+
```
Comment (Just-Bob-Dev) [inline src/todoListItem.js:6]: nit: empty space

**Review by Just-Bob-Dev — COMMENTED** [review]
Everything worked, all criteria was met and you cleared the input! Great job!

**Comment by curt4207** [comment]
> Hey Curt, while reviewing your PR I noticed your still on the lesson-1-1 branch. After you get this merged in don't forget to create a new branch for next weeks work.

yea I noticed it but forgot to fix it. thx

**Pull request merged**